sdk/src/main/java/com/mapbox/maps/MapInitOptions.kt

```diff
@@ -87,4 +67,34 @@ data class MapInitOptions constructor(
       )
     }
   }
+}
+
+/**
+ * Get a default [MapOptions.Builder] with reasterization mode [GlyphsRasterizationMode#ALL_GLYPHS_RASTERIZED_LOCALLY]
+ * @property context the context of the application.
+ * @return [MapOptions.Builder]
+ */
+fun MapOptions.Builder.applyDefaultParams(context: Context): MapOptions.Builder = also {
```

Comment:
Can we move those functions to appropriate files `MapOptions` and `ResourceOptions`? For me it looks strange why they're located here.

Reply:
MapOptions and MapOptions are generated classes, I thought this is the most relevant place to have these extension function, but we can always use new files holding these extension functions if you prefer.

Reply:
No, it's fine then I guess.